### PR TITLE
build: bump year

### PR DESCRIPTION
**Pull request by CaerusKaru**
Happy New Year :tada:

### CODE REVIEW

File: packages/language-service/bundles/banner.js.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * @license Angular v0.0.0-PLACEHOLDER
- * (c) 2010-2018 Google, Inc. https://angular.io/
+ * (c) 2010-2019 Google, Inc. https://angular.io/
```
Comment (alfaproject) [inline packages/language-service/bundles/banner.js.txt]: LLC

**Review by petebacondarwin — APPROVED** [review]

**Review by kyliau — APPROVED** [review]

File: aio/content/license.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 @description
 The MIT License
 
-Copyright (c) 2014-2018 Google, Inc.
+Copyright (c) 2014-2019 Google, Inc.
```
Comment (IgorMinar) [inline aio/content/license.md]:
```suggestion
Copyright (c) 2014-2019 Google LLC
```

**Review by IgorMinar — COMMENTED** [review]
I see a bunch of other inconsistencies in this PR. I'll make a local change and push a new fixup commit to this PR rather than nitpick all over the PR.

**Review by IgorMinar — APPROVED** [review]

File: aio/src/app/layout/footer/footer.component.html [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
-    Code licensed under an <a href="license" title="License text" >MIT-style License</a>.
-    Documentation licensed under
-    <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.
-  </p>
-  <p>
-    Version {{versionInfo?.full}}.
-  </p>
-  <!-- TODO: twitter widget (but only on pages that use twitter) -->
+<p>
+  Super-powered by Google ©2010-2019.
```
Comment (alfaproject) [inline aio/src/app/layout/footer/footer.component.html]: lol

**Comment by IgorMinar** [comment]
😉

File: aio/src/app/shared/web-worker.ts [hunk 4]
```diff
@@ -1,8 +1,10 @@
-/*
-Copyright 2016 Google Inc. All Rights Reserved.
-Use of this source code is governed by an MIT-style license that
-can be found in the LICENSE file at http://angular.io/license
-*/
+/**
```
Comment (alfaproject) [inline aio/src/app/shared/web-worker.ts]: Why is this different than `search.service.ts`?

Comment (IgorMinar) [reply aio/src/app/shared/web-worker.ts]: yeah.. I missed that. I'll push a fix

Comment (IgorMinar) [reply aio/src/app/shared/web-worker.ts]: thanks

**Comment by IgorMinar** [comment]
caretaker: no g3 impact

**Review by gkalpak — COMMENTED** [review]
OOC, @IgorMinar, why the different copyright notices? We have:

- [Copyright (c) 2010-2019 Google LLC. http://angular.io/license][1]
- [Copyright Google LLC. All Rights Reserved.][2]
- [(c) 2010-2019 Google LLC. https://angular.io/][3] (which I have no idea what it is for :grin:)
- [Copyright Google Inc. All Rights Reserved.][4] (I assume this is left untouched to avoid having to update every single source file in `packages/`)

[1]: https://github.com/angular/angular/blob/6b3bbe54a915680b1ab5c63a124bd0cefa6c57e9/LICENSE#L3
[2]: https://github.com/angular/angular/blob/6b3bbe54a915680b1ab5c63a124bd0cefa6c57e9/aio/tools/examples/shared/boilerplate/systemjs/src/systemjs.config.web.build.js#L2
[3]: https://github.com/angular/angular/blob/6b3bbe54a915680b1ab5c63a124bd0cefa6c57e9/packages/bazel/test/ng_package/example_package.golden#L94
[4]: https://github.com/angular/angular/blob/6b3bbe54a915680b1ab5c63a124bd0cefa6c57e9/packages/core/index.ts#L3

**Comment by IgorMinar** [comment]
~Oh George, I'm really disappointed that you didn't notice that there is one more:~

https://github.com/angular/angular/blob/b0f3c20a4c41710cefc4d94cbe4962a5aba2fae7/packages/core/src/di.ts#L2-L6

dang! You found that one as well! How can you be this awesome?!? You are incredible! 🙇 

I tried to decrease the number of copyright notices with dates to absolute minimum. So the first type is used only in the main license file and in the angular.io footer.

The second one is what I generally used everywhere, except for source code because I really don't want to touch hundreds if not thousands of files as we are landing all the Ivy PRs.

The third one comes from CLI - it's a golden file showing what the cli generates. I don't have control over that in this repo.

And the last one is the one I thought you missed, but of course you didn't. It's the notice in all source files in the repo. We'll need to find a good time to update these without disrupting day-to-day development too much. (it might be easier than it seems because the diff would unlikely collide, but the change is definitely out of scope for this PR).

**Pull request closed without merging**